src/src/Controller/ApiController.php

```diff
@@ -107,6 +111,38 @@ public function settingsTunnel(Request $request): JsonResponse
         return new JsonResponse($data);
     }
 
+    #[Route('/api/settings/shell', name: 'api_settings_shell')]
+    public function settingsShell(Request $request): JsonResponse
```

Comment:
This could be simplfied to

```php
    #[Route('/api/settings/shell', name: 'api_settings_shell')]
    public function settingsShell(Request $request): JsonResponse
    {
        if ($request->isMethod('post')) {
            $data = json_decode($request->getContent(), true);
            $response = [
                'status' => 'error',
                'message' => 'Invalid operation',
            ];

            if (isset($data['op'])) {
                if ('start' == $data['op']) {
                    $response = $this->shellHelper->startShell($data['timeout']);
                } elseif ('stop' == $data['op']) {
                    $response = $this->shellHelper->stopShell();
                } 
            }
        } else {
            $response = $this->shellHelper->getShellStatus();
        }

        return new JsonResponse($response);
    }
```

Reply:
Yes, will do :)